source/404.rst

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+Page not found
+==============
+
+Unfortunately, the page you were looking for is not available, this may be due to a page rename or an invalid URL. If you believe this is a mistake, please contact :doc:`support <docs/support>`. Otherwise, please try searching for the requested page in the lefthand search bar.
+
+.. image:: https://imgs.xkcd.com/comics/hamster_ball.png
+   :alt: XKCD comic 152
+
+*Image Credit: https://xkcd.com/152/*
```

Comment:
This doesn't really make sense in the context of our documentation. Maybe bring back the "Why did the chicken cross the road?"

Reply:
Was hoping for a funnier joke, the original one wasn't that funny.